mycroft/client/speech/listener.py

```diff
@@ -200,35 +199,39 @@ def _load_config(self):
             Load configuration parameters from configuration
         """
         config = ConfigurationManager.get()
+        self.config_core = config
         self._config_hash = hash(str(config))
-        lang = config.get('lang')
+        self.lang = config.get('lang')
         self.config = config.get('listener')
         rate = self.config.get('sample_rate')
         device_index = self.config.get('device_index')
 
         self.microphone = MutableMicrophone(device_index, rate)
         # FIXME - channels are not been used
         self.microphone.CHANNELS = self.config.get('channels')
-        self.mycroft_recognizer = self.create_mycroft_recognizer(rate, lang)
+        self.wakeword_recognizer = self.create_wake_word_recognizer()
         # TODO - localization
-        self.wakeup_recognizer = self.create_wakeup_recognizer(rate, lang)
-        self.remote_recognizer = ResponsiveRecognizer(self.mycroft_recognizer)
+        self.wakeup_recognizer = self.create_wakeup_recognizer()
+        self.responsive_recognizer = ResponsiveRecognizer(
+            self.wakeword_recognizer)
         self.state = RecognizerLoopState()
 
-    def create_mycroft_recognizer(self, rate, lang):
+    def create_wake_word_recognizer(self):
         # Create a local recognizer to hear the wakeup word, e.g. 'Hey Mycroft'
-        wake_word = self.config.get('wake_word').lower()
-        phonemes = self.config.get('phonemes')
-        threshold = self.config.get('threshold')
-        return PocketsphinxRecognizer(wake_word, phonemes,
-                                      threshold, rate, lang)
-
-    def create_wakeup_recognizer(self, rate, lang):
-        wake_word = self.config.get('standup_word', "wake up").lower()
-        phonemes = self.config.get('standup_phonemes', "W EY K . AH P")
-        threshold = self.config.get('standup_threshold', 1e-18)
-        return PocketsphinxRecognizer(wake_word, phonemes,
-                                      threshold, rate, lang)
+        LOG.info("creating wake word engine")
+        word = "hey mycroft"  # self.config.get("wake_word", "hey mycroft")
+        # TODO remove this, only for server settings compatibility
+        phonemes = self.config.get("phonemes")
```

Comment:
One more thing, could we add threshold backwards compatibility here just like `phonemes`? Otherwise, for some simpler wake words, without the changed threshold it will start triggering like crazy.

Reply:
done